build-pipelines-PR.yml

```diff
@@ -194,6 +195,7 @@ jobs:
       restoreArguments: -r win-x64
       selectOrConfig: config
       nugetConfigPath: src/nuget.config
+      externalEndpoints: dba0ba50-2d4e-4f12-9f5a-42a638da803b
```

Comment:
Could you tell me what this guid refers to? What's an `externalEndpoint` here?

Reply:
From a quick read of the [docs](https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/dotnet-core-cli-v2?view=azure-pipelines) it looks as if this is an alias for a [service connection](https://learn.microsoft.com/en-us/azure/devops/pipelines/library/service-endpoints?view=azure-devops&tabs=yaml) for a NuGet feed... Has this been working until now?

Reply:
I haven't tested it, will confirm if it works. Thanks for the comment.

Reply:
@peterbom Previous failing build - https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=7918956&view=logs&s=96ac2280-8cb4-5df5-99de-dd2da759617d

After adding external endpoint , passing build - https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=7919390&view=results

It is weird though why it doesn't fail if we provide external endpoint.